source/_components/notify.mailgun.markdown

```diff
@@ -22,22 +22,21 @@ The Mailgun notification service allows you to send emails via Mailgun's REST AP
 # Example configuration.yaml entry
 mailgun:
   domain: mg.example.com
-  api_key: token-XXXXXXXXX
-  sandbox: False
+  api_key: XXXXXXXXXXXXXX
 
 notify:
   - name: mailgun
-    platform: mailgun
+    platform: mailgun_me
```

Comment:
I'm confused where the change in platform has come from.

Reply:
Sorry, the intention was to have "name mailgun_me" and "platform: mailgun".  Good catch.  Is there a way I can fix this?  I'm new to github.